modules/project-docs/pages/compatibility.adoc

```diff
@@ -58,9 +58,12 @@ It is best to upgrade either the SDK or the Couchbase version you are using.
 | *✖*
 | *✖*
 | *◎*
-| *✔*
+| *◎* _(see below)_
 |===
 
+IMPORTANT: when running against Server 7.0β, Java SDK *3.1.6* is the latest compatible version.
```

Comment:
@osfameron  3.1.6 is the latest compatible is not right.

refer to this page https://www.couchbase.com/products/developer-sdk/compatibility
for instance Java is compatible with SDK versions ranging from SDK 3.0.9 through 3.1.3
Likewise add this comment as to why using the range of versions 
 : **The latest release of Couchbase SDKs include changes for the upcoming GA release. As a result before GA you will need to use a specific range of releases.**

Reply:
@AV25242 thanks - I'm confused though - the internal google doc linked to the ticket does mention 3.1.6, should I ignore that document in favour of the page you link above, in all cases?